docs/api-reference/fastforest.md

```diff
@@ -0,0 +1,31 @@
+### Training Algorithm Details
+Decision trees are non-parametric models that perform a sequence of simple tests
```

Comment:
Will this text be repeated in all decision tree trainer docs?

Reply:
Just FastForest Regression and classification.

---
In reply to: [277064468](https://github.com/dotnet/machinelearning/pull/3399#discussion_r277064468) [](ancestors = 277064468)